user_tools/src/spark_rapids_pytools/wrappers/emr_wrapper.py

```diff
@@ -212,8 +212,8 @@ class CliEmrServerlessMode:  # pylint: disable=too-few-public-methods
     @staticmethod
     def qualification(cpu_cluster: str,
                       remote_folder: str,
-                      job_arn: str,
-                      eventlogs: str = None,
+                      job_role_arn: str,
+                      eventlogs: str,
```

Comment:
No need to make changes to the serverless mode as it is going to be deprecated and dropped in the upcoming releases

Reply:
Removed the change